manifests/base/jupyter-habana-notebook-imagestream.yaml

```diff
@@ -0,0 +1,43 @@
+---
+# This Image is Deprecated and will not be displayed on the Dashboard
+# Keeping this here for satisfying upgrade requirements
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
```

Comment:
Note: I guess that having here the:
```
  labels:
    opendatahub.io/notebook-image: "false"
```
would have same effect as removing them completely?

If so, maybe next time it would be good to keep the label there so it's clear what actual label has different value... but it's not that important, just an idea.